pkg/innerring/processors/governance/process_update.go

```diff
@@ -123,7 +123,7 @@ func (gp *Processor) processAlphabetSync(txHash util.Uint256) {
 		}
 	}
 
-	// 4. Update NeoFS contract in main net.
+	// 4. Update NeoFS contract in the mainnet.
```

Comment:
i would also change all "mainnet" to "mainchain", but not sure

Reply:
Let's ask someone else, I'm not the person to decide upon it.
@fyrchik @realloc @alexvanin

Reply:
Let's keep mainnet here. Otherwise, we risk to confuse everyone with a new term.